### ISSUE TITLE
Hard to install the program probably

### ISSUE THREAD

**Issue by huy-le** [krzysztofzablocki/Sourcery#312]
Do you thing 4 ways you are using for installing this command line tools are hard to follow, undocumented?

**Comment by ilyapuchka** [comment]
Hey @huy-le . What exactly do you find hard to follow and what issues do you have installing Sourcery?

**Comment by worthbak** [comment]
I eventually got Sourcery installed properly, but the instructions were pretty unhelpful - the README notes that, for Cocoapods installation, you "just need to add" the following as a run script phase:

```
$PODS_ROOT/Sourcery/bin/sourcery {source} {templates} {output}
```

However, my project wouldn't compile until I added this:

```
"${PODS_ROOT}/Sourcery/bin/sourcery" --sources "${SRCROOT}/SourceryTests" --templates "${SRCROOT}/Templates" --output "${SRCROOT}/CodeGenerated"
```

Even the example project needed the script to be changed before it would compile. I get that `{source} {templates} {output}` is pseudocode, but it's totally unclear that needed arguments are omitted, or that the file paths need to be structured in a particular way. The README could be *much* more clear on this front.

(btw, Sourcery is otherwise an amazing tool, once it's set up property - thanks for your work!!!)

**Comment by ilyapuchka** [comment]
Thanks for your feedback @worthbak! Indeed those instructions should be more clear and they are in fact out of date with the latest versions of the tool. We will fix that.

**Comment by Antondomashnev** [comment]
Hi @huy-le @worthbak check the updated documentation in `master` and let us know if it becomes clearer 😄

**Comment by worthbak** [comment]
@Antondomashnev looks better! My only suggestion would be to provide a full example, as `--sources <sources path>` might seem a bit ambiguous to new users (especially because Sourcery makes heavy use of the `<<>>`` syntax in its regular usage). Or, perhaps just link to the relevant line in your example project's scheme - that was useful for me when getting it up and running 👍 thanks!!
